bitnami/etcd/templates/cronjob-defrag.yaml

```diff
@@ -0,0 +1,100 @@
+{{- /*
+Copyright Broadcom, Inc. All Rights Reserved.
+SPDX-License-Identifier: APACHE-2.0
+*/}}
+
+{{- if .Values.defrag.enabled }}
+apiVersion: {{ include "common.capabilities.cronjob.apiVersion" . }}
+kind: CronJob
+metadata:
+  name: {{ printf "%s-defrag" (include "common.names.fullname" .) | trunc 63 | trimSuffix "-" }}
+  namespace: {{ .Release.Namespace | quote }}
+  {{- $labels := include "common.tplvalues.merge" ( dict "values" ( list .Values.defrag.cronjob.labels .Values.commonLabels ) "context" . ) }}
+  labels: {{- include "common.labels.standard" ( dict "customLabels" $labels "context" $ ) | nindent 4 }}
+  {{- if or .Values.defrag.cronjob.annotations .Values.commonAnnotations }}
+  {{- $annotations := include "common.tplvalues.merge" ( dict "values" ( list .Values.defrag.cronjob.annotations .Values.commonAnnotations ) "context" . ) }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" $annotations "context" $) | nindent 4 }}
+  {{- end }}
+spec:
+  {{- if .Values.defrag.cronjob.startingDeadlineSeconds }}
+  startingDeadlineSeconds: {{ .Values.defrag.cronjob.startingDeadlineSeconds }}
+  {{- end }}
+  concurrencyPolicy: {{ .Values.defrag.cronjob.concurrencyPolicy }}
+  schedule: {{ .Values.defrag.cronjob.schedule | quote }}
+  suspend: {{ .Values.defrag.cronjob.suspend }}
+  successfulJobsHistoryLimit: {{ .Values.defrag.cronjob.successfulJobsHistoryLimit }}
+  failedJobsHistoryLimit: {{ .Values.defrag.cronjob.failedJobsHistoryLimit }}
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          {{- $mergedLabels := mergeOverwrite (dict) .Values.commonLabels .Values.defrag.cronjob.podLabels -}}
```

Comment:
CC: @mikyll @juan131 

The `-}}`, at the end here, messes the indentation of the following element resulting in an incorrect rednerization
```yaml
spec:
  jobTemplate:
    spec:
      template:
        metadata:labels:
          app.kubernetes.io/instance: mimir
          app.kubernetes.io/managed-by: Helm
          app.kubernetes.io/name: etcd
          app.kubernetes.io/version: 3.5.16
          helm.sh/chart: etcd-10.4.0
```

Reply:
Fixed in #30052